frontend.go

```diff
@@ -15,18 +15,22 @@
 package casbin
 
 import (
+	"bytes"
 	"encoding/json"
 )
 
 func CasbinJsGetPermissionForUser(e IEnforcer, user string) ([]byte, error) {
-	policy, err := e.GetImplicitPermissionsForUser(user)
-	if err != nil {
-		return nil, err
+	model := e.GetModel()
+	m := map[string]interface{}{}
+	m["m"] = model.ToText()
+	policies := e.GetPolicy()
+	for i := range policies {
```

Comment:
Where are g policies?

Reply:
@hsluoyz I just followed node-casbin's implementation
https://github.com/casbin/node-casbin/blob/master/src/frontend.ts

Reply:
I guess we should be consistent with node-casbin

Reply:
@Zxilly

Reply:
@closetool Yes, seems node-casbin has bug too, it cannot handle multipy role-manager.
In fact, this function means to get the whole model and all policy. You can implement it in golang style. I will later fix node-casbin version.